verify/verify.go

```diff
@@ -719,10 +728,15 @@ func setProduct(attestation *spb.Attestation, product *spb.SevProduct) {
 	attestation.Product = product
 }
 
-// fillInAttestation uses AMD's KDS to populate any empty certificate field in the attestation's
-// certificate chain.
-func fillInAttestation(attestation *spb.Attestation, options *Options) error {
-	var productOverridden bool
+func cpuidWorkaround(attestation *spb.Attestation, options *Options) (string, func([]byte) error, func() error, error) {
```

Comment:
Can you add some comments for this? What this function does is not obvious from the name

Reply:
Added.